README.md

```diff
@@ -3,24 +3,25 @@
 ![WASI](WASI.png)
 
 This repository is for the WebAssembly System Interface (WASI) Subgroup of the
-[WebAssembly Community Group]. Its [Charter] describes the goals, scope and
-deliverables of the group. The repository may contain meeting notes, reports,
-documentation, specifications and/or software produced by the group (although
-larger projects may also have their own repositories).
+[WebAssembly Community Group]. It includes:
+ - [charter]: describes the goals, scope and deliverables of the group
+ - [docs]: learn more about WASI
+ - [meetings]: schedules and agendas for video conference and in-person meetings
+ - [phases]: the current WASI specifications
+ - [proposals]: the status of each new specification proposal
 
+[charter]: Charter.md
+[docs]: docs
+[meetings]: meetings/README.md
+[phases]: phases/README.md
+[proposals]: docs/Proposals.md
 [WebAssembly Community Group]: https://www.w3.org/community/webassembly/
-[Charter]: Charter.md
 
 We'll be adding more content here before long, but for now, check out these:
  - https://hacks.mozilla.org/2019/03/standardizing-wasi-a-webassembly-system-interface/ - Blog post introducing WASI
- - https://wasi.dev/ - Links to more information about WASI, including
-   how to get started using it.
+ - https://wasi.dev/ - Links to more information about WASI, including how to get started using it.
  - https://github.com/WebAssembly/WASI/issues - This repo's issue tracker
```

Comment:
@sunfishcode: I was thinking that these links may not be super-necessary to include on the front page; what do you think about creating a `docs/README.md` and moving them there? `docs/README.md` could also set a bit of context before directing users to the more specific pages (e.g. Design Principles, High-Level Goals).

Reply:
That sounds like a good idea to me. That way the main repo README.md can focus more on being a landing page to help people find what they're looking for, while docs/README.md can be more about talking about the different documents it links to.